app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Handles static page for home and help.
+class StaticPagesController < ApplicationController
+  
+  # Get home.
+  def home;  end
```

Comment:
I don't think you need any of these methods. This can be an empty class. If I'm wrong, then at least you need to comment them properly.

Reply:
is "# Get static_pages/home." the correct commenting?

Reply:
Get -> GET
just see any of the controllers in IO/DM